docs/_docs/variables.md

```diff
@@ -186,16 +186,32 @@ following is a reference of the available data.
       </p></td>
     </tr>
     <tr>
+      <td><p><code>site.url</code></p></td>
+      <td><p>
+
+        Contains the url of your site as it is configured in the <code>_config.yml</code>.
+        For example, if you have <code>url: http://mysite.com</code>
+        in your configuration file, then in your Posts and Pages it will be
```

Comment:
`in your Posts and Pages` is too specific. Anywhere Liquid is used, the `site` variable is available. Could we just say:

> in your configuration file, then it will be accessible in Liquid as <code>site.url</code>

?